content/tutorial-regular-keys.md

```diff
@@ -0,0 +1,300 @@
+# Working with Regular Keys
+
+The XRP Ledger allows an account to authorize a secondary key pair, called a "regular key," to sign future transactions. If the private key of a regular key is compromised, you can remove or replace it without changing the rest of your account and re-establishing its relationships to other accounts. You can also rotate a regular key proactively. (Neither of those things is possible for the master key pair of an account, which is intrinsically linked to the account's address.)
```

Comment:
Great question! @mDuo13, do you have any thoughts on this? Is the compromise of a regular key a sign that your master key has also been or may soon be compromised? Is there any way in which knowing a regular private key will get you closer to knowing a master private key -- in terms of traversing info in the XRP Ledger network? It's a different story if someone broke into your safe where you stored a piece of paper with your master and regular private keys written on it. Is the strategy behind the regular key that it is the first line of defense - so if it gets compromised, then it gives you a warning sign that you need to disable your master key and set up a new account? Or is the strategy that you get up a regular key so that it is the first line of defense that takes the hit so that you can just set up a new one and you don't have to disable your master key?